docs/config/misc.md

```diff
@@ -65,6 +65,32 @@ return {
   -- Set this to "NeverPrompt" if you don't like confirming closing
   -- windows every time
   window_close_confirmation = "AlwaysPrompt",
+
+  -- When true (the default), watch the config file and reload it
+  -- automatically when it is detected as changing.
+  -- When false, you will need to manually trigger a config reload.
+  automatically_reload_config = true,
```

Comment:
Thanks for picking this up!

My plan was to break this `misc.md` file up so that there is a file per option over in https://github.com/wez/wezterm/tree/main/docs/config/lua/config
That way each option can be linked to from other parts of the docs and it is a little bit easier to search for things.

Would you mind putting these newly documented items into files in that directory instead?